### DISCUSSION TITLE
Nickel vs KCL

### DISCUSSION THREAD

**Discussion by nwmcsween** [tweag/nickel#1148] [General]
Please see: https://github.com/KusionStack/KCLVM or https://kcl-lang.io it seems both nickel and KCL overlap quite a bit.

**Comment by yannham** [comment]
Hello, thanks for the reference! I got aware of KCL quite recently to be honest, and haven't had the time to play with it more. However, it does seem to be quite close to Nickel - at least, in the philosophical approach (typing, modularity, etc.). I'll update the RATIONALE and various comparisons as we get more insights on KCL.
